cicd/csm_web.spec

```diff
@@ -58,8 +56,6 @@ systemctl stop csm_web
 %postun
 [ $1 -eq 1 ] && exit 0
 rm -f /usr/bin/csm_web 2> /dev/null;
-rm -rf /etc/systemd/system/csm_web.service 2> /dev/null;
```

Comment:
@avrnv Please make the same changes as you have done for csm agent PR i.e checking of service file first and then performing the steps

Reply:
Done.